pyrit/datasets/orchestrators/tree_of_attacks/initial_prompt.yaml

```
@@ -0,0 +1,13 @@
+name: tree_of_attacks_with_pruning_initial_prompt
+description: Based on the paper 'Tree of Attacks Jailbreaking Black-Box LLMs Automatically' https://arxiv.org/abs/2312.02119
+harm_category: NA
+author: Anay Mehrotra, Manolis Zampetakis, Paul Kassianik, Blaine Nelson, Hyrum Anderson, Yaron Singer, Amin Karbasi

```

Nit: Format in authors listing. I see both formats but one author per line is more common that I've seen in YAML templates.
```suggestion
author: 
  - Anay Mehrotra
  - Manolis Zampetakis
  - Paul Kassianik
  - Blaine Nelson 
  - Hyrum Anderson
  - Yaron Singer
  - Amin Karbasi
```

---

I'm not opposed to that in principle but it would require changing our data model for how we treat metadata. I don't want to do that in this PR but we can create an issue for that.

---

#282